spec/index.html

```diff
@@ -626,6 +551,66 @@ <h2>N-Quads Internet Media Type, File Extension and Macintosh File Type </h2>
   </dl>
 </section>
 
+<section id="section-ack" class="informative">
+  <h2>Acknowledgments</h2>
+
+  <section class="informative">
+    <h3>Acknowledgments for RDF 1.1</h3>
+    <p>The editor of the RDF 1.1 edition acknowledges valuable
+      contributions from Gregg Kellogg, Andy Seaborne, Eric
+      Prud'hommeaux, Dave Beckett, David Robillard, Gregory Williams,
+      Antoine Zimmermann, Sandro Hawke, Richard Cyganiak, Pat Hayes,
+      Henry S. Thompson, Bob Ferris, Henry Story, Andreas Harth, Lee
+      Feigenbaum, Peter Ansell, Evan Patton and David Booth.</p>
+    <p>This specification is a product of extensive deliberations by the
+      members of the RDF Working Group chaired by Guus Schreiber and David Wood.
+      It draws upon the earlier specification in <em><a href="http://sw.deri.org/2008/07/n-quads/">N-Quads: Extending N-Triples with Context</a></em>,
+      edited by Richard Cyganiak, Andreas Harth, and Aidan Hogan.</p>
+  </section>
+
+  <section class="informative">
+    <h3>Acknowledgments for RDF 1.2</h3>
+
+    <p>The editors of the RDF 1.2 edition acknowledge valuable contributions from Andy Seabourn.</p>
```

Comment:
```suggestion
    <p>The editors of the RDF 1.2 edition acknowledge valuable contributions from Andy Seaborne.</p>
```

Reply:
Oops! I'll fix a similar problem in N-Triples.